docs/handbook/community.md

```diff
@@ -34,9 +34,15 @@ Link to PDF Document: <a href= "../pdf/HowToInstallOracleDB.pdf" target="_blank"
 <p>Unfortunately, it is not possible to run Oracle Database version 12 and above on OpenIndiana.</p>
 </div>
 
+## How to Install Sun Ray Server on Openindiana Hipster
```

Comment:
```suggestion
## How to Install Sun Ray Server Software on OpenIndiana Hipster
```

Reply:
or Sun Ray Software, it's the name used in Oracle docs

Reply:
Right, lets use whatever official/well-known name is out there and use it consistently.